xmrstak/cli/cli-miner.cpp

```diff
@@ -72,7 +72,9 @@ void help()
 #ifdef _WIN32
 	cout<<"  --noUAC                    disable the UAC dialog"<<endl;
 #endif
-	cout<<"  --benchmark BLOCKVERSION   ONLY do a 60-second benchmark and exit"<<endl;
+	cout<<"  --benchmark BLOCKVERSION   ONLY do a benchmark and exit"<<endl;
+	cout<<"  --benchwait WAIT_SEC       ... benchmark wait time (default 30)"<<endl;
```

Comment:
Please remove `(default ...)` in both options. The reason is that normally a default value is showing the default for the option if no argument is given. This is not supported by xmr-stak. To not confuse the user it is much better to remove the default value.